nio_command_block.py

```diff
@@ -0,0 +1,173 @@
+from nio.metadata.properties import IntProperty, ListProperty, \
+    SelectProperty, ObjectProperty, PropertyHolder, StringProperty, \
+    TimeDeltaProperty, ExpressionProperty
+from nio.common.block.base import Block
+from nio.common.discovery import Discoverable, DiscoverableType
+from nio.common.signal.base import Signal
+from nio.modules.scheduler import Job
+from urllib.parse import urlencode
+import base64
+from .oauth2_mixin.oauth2 import OAuth2, OAuth2Exception
+import requests
+from enum import Enum
```

Comment:
If this is the only issue it's nbd but in general, we want global imports at the top and local ones at the bottom.
See the [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports) for more